api/Navigator.json

```diff
@@ -4218,6 +4218,8 @@
       },
       "unregisterProtocolHandler": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/unregisterProtocolHandler",
+          "spec_url": "https://html.spec.whatwg.org/multipage/system-state.html#custom-handlers",
```

Comment:
```suggestion
          "spec_url": "https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-unregisterprotocolhandler-dev",
```

Reply:
Thanks for the suggestion @queengooborg !